libevmcore/Instruction.cpp

```diff
@@ -82,6 +82,8 @@ const std::map<std::string, Instruction> dev::eth::c_instructions =
 	{ "SSTORE", Instruction::SSTORE },
 	{ "JUMP", Instruction::JUMP },
 	{ "JUMPI", Instruction::JUMPI },
+	{ "JUMPV", Instruction::JUMP },
```

Comment:
Yes, they have to look like instructions to the interpreter, not just numbers.  The only place they are used outside of the interpreter is tracing and debugging via onOperation().   There is a comment in Instruction.h, and I just put them inside macros:
```
#ifdef EVM_USE_SYNTHETIC_INSTRUCTIONS
	// these are generated by the interpreter - should never be in user code
	PUSHC = 0xad,		///< push value from constant pool
	JUMPV,				///< alter the program counter - pre-verified
	JUMPVI,				///< conditionally alter the program counter - pre-verified
#endif

```